packages/angular/src/app/components/status-select-box/status-select-box.component.html

```diff
@@ -0,0 +1,27 @@
+<dx-select-box
+  label="Status"
+  [value]="value"
+  (valueChange)="valueChange.emit($event)"
+  [items]="items"
+  [readOnly]="readOnly"
+  [stylingMode]="stylingMode"
+  [labelMode]="labelMode"
+  width="100%"
+  fieldTemplate="field"
+>
+  <div *dxTemplate="let data of 'field'" class="status-editor-field">
+    <contact-status class="status-indicator"
+                    [showText]="false"
+                    [value]="data"></contact-status>
```

Comment:
Let's change formatting:

```suggestion
    <contact-status class="status-indicator"
                    [showText]="false"
                    [value]="data">
    </contact-status>
```